Source/WebKit/UIProcess/API/Cocoa/WKPreferences.mm

```diff
@@ -1019,12 +1033,12 @@ - (BOOL)_pageCacheSupportsPlugins
 
 - (void)_setShouldPrintBackgrounds:(BOOL)enabled
 {
-    _preferences->setShouldPrintBackgrounds(enabled);
+    [self setShouldPrintBackgrounds:enabled];
 }
 
 - (BOOL)_shouldPrintBackgrounds
 {
-    return _preferences->shouldPrintBackgrounds();
+    return [self shouldPrintBackgrounds];
```

Comment:
Nit: This can use dot notation.

Reply:
should `setShouldPrintBackgrounds` also use dot notation?

Reply:
That would be good too. My understanding is that we prefer dot notation when possible (the notable exception being when using `RetainPtr`).

Reply:
also would It be `    return self.shouldPrintBackgrounds;` or without the `self`?

Reply:
`return self.shouldPrintBackgrounds`